src/bundles/notify.js

```diff
@@ -84,6 +101,12 @@ const notify = {
       if (eventId === 'STATS_FETCH_FAILED') {
         return provider === 'window.ipfs' ? 'windowIpfsRequestFailed' : 'ipfsApiRequestFailed'
       }
+      if (eventId === 'IPFS_CONNECT_FAILED') {
+        return 'ipfsApiRequestFailed';
+      }
+      if (eventId === 'IPFS_CONNECT_SUCCEED') {
+        return 'ipfsIsBack';
+      }
```

Comment:
These two errors are recycled. 
The english text for each are respectively:
- 'Could not connect. Please check if your daemon is running.'
- 'Normal IPFS service has resumed. Enjoy!'

I'd create a custom error message, but I can only responsibly source French, English, and maybe Chinese translations. Ideally, the messages would say something like:
- 'Unable to connect to the provided IPFS API address'
- 'Sucessfully connected to the IPFS API address'

Thoughts?

Reply:
Custom errors are good! We use Transifex to source internationalization and have an active network of community translators, so only need English.